Assets/Editor/UnitTests/Models/SchedulerEditorTest.cs

```diff
@@ -38,20 +38,18 @@ public void Init()
         if (FunctionsManager.ScheduledEvent == null)
         {
             new FunctionsManager();
-            FunctionsManager.ScheduledEvent.LoadScript(LuaFunctionString, "ScheduledEvent");
         }
 
+        FunctionsManager.ScheduledEvent.LoadScript(LuaFunctionString, "ScheduledEvent");
+
         if (PrototypeManager.SchedulerEvent == null)
         {
             new PrototypeManager();
```

Comment:
Haaa sorry, this is not a stored variable but a static access, this is for another PR but we should really get rid of this for a singleton.

It's good the way it is for now.